### PR TITLE
bring galaxy up to spec, ketarin-ize it

### PR DESCRIPTION
**Pull request by Link-Satonaka**
Also updates the icon.
I can't update iconUrl since rawgit needs commit urls :)

If you don't want to keep updating this package, add me as a maintainer on chocolatey: LinkSatonaka

### CODE REVIEW

**Comment by Link-Satonaka** [comment]
though imo this package should be renamed to gog-galaxy

**Comment by AnthonyMastrean** [comment]
I'm happy to make you a maintainer. You can submit `gog-galaxy` and make it depend on this package, with no (un)install script. Chocolatey really needs to support aliases!

Do you have a Ketarin server/scheduled-task running? Would it be easier if you're running Ketarin to own the repo? If so, we can migrate the code and coordinate shutting down the copy of the package in this repo.

(I've found it nice to have the package in 1 repo, especially if the owner has Ketarin running)

**Comment by AnthonyMastrean** [comment]
Oh and I used the image that you see now because it was higher resolution. I also found the one you did, but it's teensy weensy! ;)

File: packages/goggalaxy/goggalaxy.nuspec [hunk 1]
```diff
@@ -1,19 +1,33 @@
...
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <!--<iconUrl>https://cdn.rawgit.com/AnthonyMastrean/chocolateypackages/a561f0de0d19224bf85d694f74f6ce52f8bf9c08/public/icons/goggalaxy.png</iconUrl>-->
+    <!--<dependencies>
+      <dependency id="" version="__VERSION__" />
+      <dependency id="" />
+    </dependencies>-->
+    <releaseNotes>https://www.gog.com/galaxy#glx-whats-new</releaseNotes>
+    <!--<provides></provides>-->
+  </metadata>
+  <files>
```
Comment (AnthonyMastrean) [inline packages/goggalaxy/goggalaxy.nuspec]: I don't know why this is in the Chocolatey package template, it's default behavior to include everything in the folder containing the nuspec file. Blah, it's not your fault.

File: packages/goggalaxy/tools/chocolateyInstall.ps1 [hunk 2]
```diff
@@ -1,6 +1,8 @@
-﻿$id     = "goggalaxy"
-$url    = "http://cdn.gog.com/open/galaxy/client/installers/setup_galaxy_1.0.2.958.exe"
-$kind   = "EXE"
-$silent = "/VERYSILENT /NORESTART"
-
-Install-ChocolateyPackage -PackageName $id -FileType $kind -Silent $silent -Url $url
+﻿$packageName = 'GOGGalaxy'
+$version = '{{PackageVersion}}'
+$url = "http://cdn.gog.com/open/galaxy/client/setup_galaxy_1.1.5.28.exe"
```
Comment (AnthonyMastrean) [inline packages/goggalaxy/tools/chocolateyInstall.ps1]: To fully automate this package, you'll need to incorporate the `{{PackageVersion}}` into this URL, right?

**Comment by Link-Satonaka** [comment]
I'll take a look at this soon- yeah the image is tiny, but the bigger one is the GOG logo, not the goggalaxy logo! It probably doesn't matter that much- I was thinking of submitting the correct image and seeing how awful it looks, then reverting. But if you want to prioritize higher res, I can just leave the old one in.

**Comment by Natrezim** [comment]
Hellou @Link-Satonaka, @AnthonyMastrean 
Is this moving somewhere? I would like to update goggalaxy to latest version 1.1.30.5. Is there anything I can help? I already fork this and updated scripts.

**Pull request closed without merging**